themes/custom/az_barrio/css/style.css

```diff
@@ -399,16 +399,21 @@ padding: 0.625rem .75rem;
   height: 43px;
 }
 
-.search-block-form .input-group-append button {
+form.search-block-form .input-group .input-group-append button:hover,
+form.search-block-form .input-group .input-group-append button:focus {
+    background-color: #8B0015;
+    color: #ffffff;
+}
+
+.az-search-block .search-block-form .input-group-append button {
```

Comment:
```suggestion
.az-search-block form.search-block-form .input-group .input-group-append button {
```

Reply:
Yeah I think this and the other suggest change look good.

Reply:
Thanks!